hydroengine_service/main.py

```diff
@@ -1202,74 +1202,80 @@ def get_liwo_scenarios():
         'velocity': 'b2',
         'riserate': 'b3',
         'damage': 'b4',
-        'casualties': 'b5'
+        'fatalities': 'b5'
     }
 
     assert band in bands
 
     styles  = {
         'waterdepth': {
-            'min': 0,
-            'max': 6,
-            'palette': ','.join([
-                '#00c3ff',
-                '#00c3ff',
-                '#006fff',
-                '#0043a8',
-                '#002673',
-                '#4d0073',
-                '#73004c'
-            ])
+            'sld_style': '\
```

Comment:
python has [triple quote](https://docs.python.org/3/tutorial/introduction.html#strings) strings which allow you to avoid the multiple `\`.